### PR TITLE
[MAIN-6] Edit FeedForward

### PR DESCRIPTION
**Pull request by flukekd**
# Pull Request

Story ID: MAIN-6
Title: Edit FeedForward
Card URL: https://trello.com/c/tzaqPFER
Author: @flukekd

### CODE REVIEW

**Comment by flukekd** [comment]
Hyperparameters:
batch_size: 16
block_size: 1024
n_embd: 384
n_head: 6
n_layer: 6
dropout: 0.1
vocab_size: 92

Training steps: 5000

Output after changes:
เคยเห็นทีมตัวเองเล่นรอบสุดท้ายเวิลด์คัพมาตลอด แบบนี้พวกเขาจะทำอะไรในช่วงซัมเมอร์ปีหน้า เมื่อไม่มีบอลให้เชียร์<h>เห็นที่ทำอะไรแข่งขันอยู่ในนี้ จ.เย็นที่ในปีหน้ามีคนสำคัญคง

**Review by cuberetry — APPROVED** [review]
lgtm
